queries/coingecko/v1/tickers.sql

```diff
@@ -466,10 +465,11 @@ xyk_pools AS (
       0 as liquidity_in_usd, 
       MAX(amount_in / amount_out) OVER (PARTITION BY asset_in) as high,
       MIN(amount_in / amount_out) OVER (PARTITION BY asset_in) as low,
-      rn
+      ROW_NUMBER() OVER (PARTITION BY tm.id, tme.id ORDER BY timestamp DESC) as rn
   FROM xyk_ranking xr
   JOIN token_metadata tm ON xr.asset_in = tm.id::text
   JOIN token_metadata tme ON xr.asset_out = tme.id::text
+  WHERE tm.id < 100 AND tme.id < 100
```

Comment:
what does this do?

Reply:
> what does this do?

added a comment, this excluded assets like HDXb, 4Pool etc from reporting